ts/sagas/startup.ts

```diff
@@ -44,6 +44,7 @@ import { watchLogoutSaga } from "./startup/watchLogoutSaga";
 import { watchPinResetSaga } from "./startup/watchPinResetSaga";
 import { watchSessionExpiredSaga } from "./startup/watchSessionExpiredSaga";
 import { watchWalletSaga } from "./wallet";
+import walletSaga from "./wallet";
```

Comment:
as we are also have non-default exports (`watchWalletSaga`) I suggest to avoid default exports (`walletSaga`)